### PR TITLE
Minimal watchdog support

### PR DESCRIPTION
**Pull request by jacobmcnamee**
- Update U-Boot to enable watchdog during boot for prod config
- Add init script to disable watchdog in Linux

TODO:
- [x] Update U-Boot https://github.com/swift-nav/u-boot-xlnx/pull/26

/cc @swift-nav/firmware

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0.0-branch-41-g50d898e
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-41-g50d898e/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.0.0-branch-41-g50d898e/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-41-g50d898e

File: board/piksiv3/rootfs-overlay/etc/init.d/S95watchdog_disable [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+name="watchdog_disable"
+
+start() {
+  printf "V" > /dev/watchdog
+}
+
+stop() {
+
```
Comment (jacobmcnamee) [inline board/piksiv3/rootfs-overlay/etc/init.d/S95watchdog_disable]: `/etc/init.d/S95watchdog_disable: line 11: syntax error: unexpected "}"`

Need https://github.com/swift-nav/piksi_buildroot/blob/master/board/piksiv3/rootfs-overlay/etc/init.d/S84piksi_fpga#L18

**Comment by jacobmcnamee** [comment]
Bench tested and ready to merge after U-Boot update

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0.0-branch-44-ge733aa6
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-44-ge733aa6/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.0.0-branch-44-ge733aa6/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-44-ge733aa6

**Comment by anonymous-travis-swiftnav** [comment]
## v1.0.0-branch-53-g826f4da
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-53-g826f4da/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.0.0-branch-53-g826f4da/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.0.0-branch-53-g826f4da

**Pull request merged**